Makefile

```diff
@@ -19,41 +16,14 @@ clean:
 
 .PHONY: check-encoding
 check-encoding:
-	! find pkg examples -name "*.go" -type f -exec file "{}" ";" | grep CRLF
+	! find . -name "*.go" -type f -exec file "{}" ";" | grep CRLF
```

Comment:
Given that later we call `go mod vendor`, this could search all of the vendored files as well. If you can exclude `vendor/` from it, that would work.

Reply:
Good catch. I've excluded `vendor` in `find` in the updated commit be07485.